relay/sources/relayd/Cargo.toml

```diff
@@ -48,6 +48,16 @@ reqwest = "0.9"
 chrono = { version = "0.4", features = ["serde"] }
 regex = "1.1"
 itertools = "0.8"
+tokio-process = "0.2.3"
+tokio-io = "0.1.12"
+hyper = "0.12.31"
+sha1 = "0.6.0"
+sha2 = "0.8.0"
+io = "0.0.2"
```

Comment:
This crate seems to only contain https://docs.rs/io/0.0.2/src/io/lib.rs.html#1-3 so I guess it is not what you expected :)